railties/lib/rails/tasks/framework.rake

```diff
@@ -48,7 +48,7 @@ namespace :app do
           require 'rails/generators'
           require 'rails/generators/rails/app/app_generator'
           gen = Rails::Generators::AppGenerator.new ["rails"],
-                                                    { api: !!Rails.application.config.api_only },
+                                                    { api: !!Rails.application.config.api_only, update: true, force: ENV['FORCE'] },
```

Comment:
Yes. This is only for tests. The other option was manually create the generator class and pass `update: true` but I chose to call `bin/rails app:update` directly from tests.

In the long term, I would definitely like to move the update task to new command line infrastructure :)